plugins/k8s/resoto_plugin_k8s/resources.py

```diff
@@ -1300,7 +1312,7 @@ class KubernetesPersistentVolumeSpec:
         "vsphere_volume": S("vsphereVolume"),
     }
     access_modes: List[str] = field(factory=list)
-    aws_elastic_block_store: Optional[str] = field(default=None)
+    aws_elastic_block_store: Optional[KubernetesPersistentVolumeSpecAwsElasticBlockStore] = field(default=None)
```

Comment:
This is the generic k8s collector: k8s can be installed inside aws, gcp or any other provider.
For that reason we can not change this property to something AWS specific.

Reply:
After pressing request changes, I realised, that `aws_elastic_block_store` is very aws specific ;) - I take back my comment.